From cbb76f2ffb934ec308d087be629e91a66898b6fc Mon Sep 17 00:00:00 2001
From: Yan Markman <ymarkman@marvell.com>
Date: Wed, 3 Jan 2018 20:54:42 +0200
Subject: [PATCH 2198/2241] dts: cleanup not supported vanilla

"vanilla" is not supported on Kernel-4.4.52
Cleanup *vanilla*

Change-Id: I0f6cfd48eec0eb7d9983e8410656ff0ef25d2b54
Signed-off-by: Yan Markman <ymarkman@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/48508
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/boot/dts/marvell/Makefile               |   3 -
 .../dts/marvell/armada-7040-db-pp2-vanilla.dts     | 205 -------------
 .../dts/marvell/armada-8040-db-pp2-vanilla.dts     | 270 -----------------
 .../dts/marvell/armada-8040-mcbin-pp2-vanilla.dts  | 321 ---------------------
 tools/configs/pp2-vanilla                          |   2 -
 5 files changed, 801 deletions(-)
 delete mode 100644 arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
 delete mode 100644 arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
 delete mode 100644 arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts
 delete mode 100644 tools/configs/pp2-vanilla

diff --git a/arch/arm64/boot/dts/marvell/Makefile b/arch/arm64/boot/dts/marvell/Makefile
index 2830fd0..0186a96 100644
--- a/arch/arm64/boot/dts/marvell/Makefile
+++ b/arch/arm64/boot/dts/marvell/Makefile
@@ -21,9 +21,6 @@ dtb-$(CONFIG_ARCH_MVEBU) += armada-7020-amc.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-70x0-customer.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-7040-pcac.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-8040-db.dtb
-dtb-$(CONFIG_ARCH_MVEBU) += armada-7040-db-pp2-vanilla.dtb
-dtb-$(CONFIG_ARCH_MVEBU) += armada-8040-db-pp2-vanilla.dtb
-dtb-$(CONFIG_ARCH_MVEBU) += armada-8040-mcbin-pp2-vanilla.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-8040-db-B.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-8040-db-C.dtb
 dtb-$(CONFIG_ARCH_MVEBU) += armada-8040-db-D.dtb
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
deleted file mode 100644
index 1890715..0000000
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-pp2-vanilla.dts
+++ /dev/null
@@ -1,205 +0,0 @@
-/*
- * Copyright (C) 2016 Marvell Technology Group Ltd.
- *
- * This file is dual-licensed: you can use it either under the terms
- * of the GPLv2 or the X11 license, at your option. Note that this dual
- * licensing only applies to this file, and not this project as a
- * whole.
- *
- *  a) This library is free software; you can redistribute it and/or
- *     modify it under the terms of the GNU General Public License as
- *     published by the Free Software Foundation; either version 2 of the
- *     License, or (at your option) any later version.
- *
- *     This library is distributed in the hope that it will be useful,
- *     but WITHOUT ANY WARRANTY; without even the implied warranty of
- *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *     GNU General Public License for more details.
- *
- * Or, alternatively,
- *
- *  b) Permission is hereby granted, free of charge, to any person
- *     obtaining a copy of this software and associated documentation
- *     files (the "Software"), to deal in the Software without
- *     restriction, including without limitation the rights to use,
- *     copy, modify, merge, publish, distribute, sublicense, and/or
- *     sell copies of the Software, and to permit persons to whom the
- *     Software is furnished to do so, subject to the following
- *     conditions:
- *
- *     The above copyright notice and this permission notice shall be
- *     included in all copies or substantial portions of the Software.
- *
- *     THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
- *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
- *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
- *     NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
- *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
- *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
- *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
- *     OTHER DEALINGS IN THE SOFTWARE.
- */
-
-/*
- * Device Tree file for Marvell Armada 7040 development board
- * This board file supports the default (A)  configuration of the board
- */
-
-#include "armada-7040-db.dtsi"
-
-/ {
-	model = "Marvell Armada-7040 development board default (A) setup - pp2-vanilla";
-	compatible = "marvell,armada7040-db-default", "marvell,armada7040-db", "marvell,armada7040",
-			"marvell,armada-ap806-quad", "marvell,armada-ap806";
-	#address-cells = <2>;
-	#size-cells = <2>;
-
-	chosen { };
-
-	aliases {
-		ethernet0 = &cp0_eth0;
-		ethernet1 = &cp0_eth1;
-		ethernet2 = &cp0_eth2;
-	};
-
-	ap806 {
-		config-space {
-			sdhci@6e0000 {
-				status = "okay";
-			};
-			serial@512000 {
-				status = "okay";
-			};
-			i2c@511000 {
-				status = "disabled";
-				clock-frequency = <100000>;
-			};
-		};
-	};
-
-	cp0 {
-		config-space {
-			/delete-node/ppv22@000000;
-			serial@702000 {
-				status = "disabled";
-			};
-			serial@702100 {
-				status = "disabled";
-			};
-			serial@702200 {
-				status = "disabled";
-			};
-			serial@702300 {
-				status = "disabled";
-			};
-			cp0_sata: sata@540000 {
-				status = "okay";
-
-				sata-port@0 {
-					status = "disabled";
-				};
-				sata-port@1 {
-					status = "okay";
-				};
-			};
-			cp0_usb3h0: usb3@500000 {
-				status = "okay";
-			};
-			cp0_usb3h1: usb3@510000 {
-				status = "okay";
-			};
-			u3d@520000 {
-				status = "disabled";
-			};
-			udc@524100 {
-				status = "disabled";
-			};
-			sdhci@780000 {
-				status = "okay";
-			};
-			spi@700680 {
-				status = "okay";
-			};
-			cp0_i2c0: i2c@701000 {
-				status = "okay";
-				clock-frequency = <100000>;
-			};
-			cp0_i2c1: i2c@701100 {
-				status = "disabled";
-			};
-
-			mdio@12a200 {
-				#address-cells = <1>;
-				#size-cells = <0>;
-				cp0_phy2: ethernet-phy@2 {
-					reg = <0>;
-				};
-				cp0_phy3: ethernet-phy@3 {
-					reg = <1>;
-				};
-			};
-			ppv22_vanilla@000000 {
-				compatible = "marvell,armada-7k-pp22";
-				reg = <0x0 0x100000>, <0x129000 0xb000>;
-				clocks = <&cp0_syscon0 1 3>, <&cp0_syscon0 1 9>, <&cp0_syscon0 1 5>;
-				clock-names = "pp_clk", "gop_clk", "mg_clk";
-				dma-coherent;
-				status = "okay";
-				cp0_eth0: eth0@010000 {
-					phy-mode = "sfi";
-					interrupts = <ICU_GRP_NSR 39 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 43 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 47 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 51 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 55 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 129 IRQ_TYPE_LEVEL_HIGH>;
-					interrupt-names = "tx-cpu0", "tx-cpu1", "tx-cpu2",
-							  "tx-cpu3", "rx-shared", "link";
-					mac-address = [00 00 00 00 00 01];
-					port-id = <0>; /* pp2_port_id */
-					gop-port-id = <0>;
-					status = "okay";
-				};
-				cp0_eth1: eth1@020000 {
-					phy-mode = "sgmii"; /* lane-0 */
-					interrupts = <ICU_GRP_NSR 40 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 44 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 48 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 52 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 56 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 128 IRQ_TYPE_LEVEL_HIGH>;
-					interrupt-names = "tx-cpu0", "tx-cpu1", "tx-cpu2",
-							  "tx-cpu3", "rx-shared", "link";
-					mac-address = [00 00 00 00 00 02];
-					port-id = <1>; /* pp2_port_id */
-					gop-port-id = <2>;
-					phy = <&cp0_phy2>;
-					status = "okay";
-				};
-				cp0_eth2: eth2@030000 {
-					phy-mode = "rgmii-id"; /* rgmii-1 */
-					interrupts = <ICU_GRP_NSR 41 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 45 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 49 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 53 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 57 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 127 IRQ_TYPE_LEVEL_HIGH>;
-					interrupt-names = "tx-cpu0", "tx-cpu1", "tx-cpu2",
-							  "tx-cpu3", "rx-shared", "link";
-					mac-address = [00 00 00 00 00 03];
-					port-id = <2>; /* pp2_port_id */
-					gop-port-id = <3>;
-					phy = <&cp0_phy3>;
-					status = "okay";
-				};
-			};
-			comphy {
-				status = "disabled";
-			};
-		};
-
-		pcie2@640000 {
-			status = "okay";
-		};
-	};
-};
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
deleted file mode 100644
index fee4b57..0000000
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-pp2-vanilla.dts
+++ /dev/null
@@ -1,270 +0,0 @@
-/*
- * Copyright (C) 2016 Marvell Technology Group Ltd.
- *
- * This file is dual-licensed: you can use it either under the terms
- * of the GPLv2 or the X11 license, at your option. Note that this dual
- * licensing only applies to this file, and not this project as a
- * whole.
- *
- *  a) This library is free software; you can redistribute it and/or
- *     modify it under the terms of the GNU General Public License as
- *     published by the Free Software Foundation; either version 2 of the
- *     License, or (at your option) any later version.
- *
- *     This library is distributed in the hope that it will be useful,
- *     but WITHOUT ANY WARRANTY; without even the implied warranty of
- *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *     GNU General Public License for more details.
- *
- * Or, alternatively,
- *
- *  b) Permission is hereby granted, free of charge, to any person
- *     obtaining a copy of this software and associated documentation
- *     files (the "Software"), to deal in the Software without
- *     restriction, including without limitation the rights to use,
- *     copy, modify, merge, publish, distribute, sublicense, and/or
- *     sell copies of the Software, and to permit persons to whom the
- *     Software is furnished to do so, subject to the following
- *     conditions:
- *
- *     The above copyright notice and this permission notice shall be
- *     included in all copies or substantial portions of the Software.
- *
- *     THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
- *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
- *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
- *     NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
- *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
- *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
- *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
- *     OTHER DEALINGS IN THE SOFTWARE.
- */
-
-/*
- * Device Tree file for Marvell Armada 8040 development board
- * This file supports option A configuration
- * CP0_RGMII1,CP1_RGMII0,CP0_EMMC,CP1_SPI(BOOT),CP1_UART,CP0_I2C,AP_eMMC,AP_UART
- */
-
-#include "armada-8040-db.dtsi"
-
-/ {
-	model = "Marvell Armada-8040 development board A setup - pp2-vanilla";
-	compatible = "marvell,armada8040-db-A", "marvell,armada8040-db", "marvell,armada8040",
-			"marvell,armada-ap806-quad", "marvell,armada-ap806";
-	#address-cells = <2>;
-	#size-cells = <2>;
-
-	chosen { };
-
-	aliases {
-		ethernet0 = &cp0_eth0;
-		ethernet1 = &cp0_eth2;
-		ethernet2 = &cp1_eth0;
-		ethernet3 = &cp1_eth1;
-	};
-
-	ap806 {
-		config-space {
-			serial@512000 {
-				status = "okay";
-			};
-			sdhci@6e0000 {
-				status = "okay";
-			};
-			i2c@511000 {
-				status = "disabled";
-			};
-			spi0: spi@510600 {
-				status = "disabled";
-			};
-		};
-	};
-
-	cp0 {
-		config-space {
-			/delete-node/ppv22@000000;
-			sata@540000 {
-				status = "okay";
-				sata-port@0 {
-					status = "okay";
-				};
-				sata-port@1 {
-					status = "okay";
-				};
-			};
-			/* usb3h0 is USB2 only (UTMI) */
-			cp0_usb3h0: usb3@500000 {
-				status = "okay";
-			};
-			cp0_usb3h1: usb3@510000 {
-				status = "okay";
-			};
-			sdhci@780000 {
-				status = "okay";
-			};
-			i2c@701000 {
-				clock-frequency = <100000>;
-				pinctrl-names = "default";
-				pinctrl-0 = <&cp0_i2c0_pins>;
-				status = "okay";
-			};
-			mdio@12a200 {
-				#address-cells = <1>;
-				#size-cells = <0>;
-				status = "okay";
-				cp0_phy1: ethernet-phy@1 {
-					reg = <1>;
-				};
-			};
-			ppv22_vanilla@000000 {
-				compatible = "marvell,armada-7k-pp22";
-				reg = <0x0 0x100000>, <0x129000 0xb000>;
-				clocks = <&cp0_syscon0 1 3>, <&cp0_syscon0 1 9>, <&cp0_syscon0 1 5>;
-				clock-names = "pp_clk", "gop_clk", "mg_clk";
-				pinctrl-names = "default";
-				pinctrl-0 = <&cp1_ge1_rgmii_pins>;
-				dma-coherent;
-				status = "okay";
-				cp0_eth0: eth0@010000 {
-					interrupts = <ICU_GRP_NSR 39 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 43 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 47 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 51 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 55 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 129 IRQ_TYPE_LEVEL_HIGH>;
-					interrupt-names = "tx-cpu0", "tx-cpu1", "tx-cpu2",
-							  "tx-cpu3", "rx-shared", "link";
-					mac-address = [00 00 00 00 00 01];
-					port-id = <0>; /* pp2_port_id */
-					gop-port-id = <0>;
-					phy-mode = "sfi";
-					status = "okay";
-				};
-				cp0_eth1: eth1@020000 {
-					interrupts = <ICU_GRP_NSR 40 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 44 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 48 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 52 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 56 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 128 IRQ_TYPE_LEVEL_HIGH>;
-					interrupt-names = "tx-cpu0", "tx-cpu1", "tx-cpu2",
-							  "tx-cpu3", "rx-shared", "link";
-				mac-address = [00 00 00 00 00 02];
-					status = "disabled";
-				};
-				cp0_eth2: eth2@030000 {
-					interrupts = <ICU_GRP_NSR 41 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 45 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 49 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 53 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 57 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 127 IRQ_TYPE_LEVEL_HIGH>;
-					interrupt-names = "tx-cpu0", "tx-cpu1", "tx-cpu2",
-							  "tx-cpu3", "rx-shared", "link";
-					mac-address = [00 00 00 00 00 03];
-					port-id = <2>; /* pp2_port_id */
-					gop-port-id = <3>;
-					phy-mode = "rgmii-id";
-					phy = <&cp0_phy1>;
-					status = "okay";
-				};
-			};
-			comphy {
-				status = "disabled";
-			};
-		};
-		pcie0@600000 {
-			status = "okay";
-		};
-		pcie2@640000 {
-			status = "okay";
-		};
-	};
-	cp1 {
-		config-space {
-			/delete-node/ppv22@000000;
-			/* cp1_usb3h0 is USB2 only (UTMI) */
-			cp1_usb3h0: usb3@500000 {
-				status = "okay";
-			};
-			sata@540000 {
-				status = "okay";
-				sata-port@0 {
-					status = "okay";
-				};
-				sata-port@1 {
-					status = "okay";
-				};
-			};
-			mdio@12a200 {
-				#address-cells = <1>;
-				#size-cells = <0>;
-				status = "okay";
-				cp1_phy1: ethernet-phy@1 {
-					reg = <0>;
-				};
-			};
-			ppv22_vanilla@000000 {
-				compatible = "marvell,armada-7k-pp22";
-				reg = <0x0 0x100000>, <0x129000 0xb000>;
-				clocks = <&cp1_syscon0 1 3>, <&cp1_syscon0 1 9>, <&cp1_syscon0 1 5>;
-				clock-names = "pp_clk", "gop_clk", "mg_clk";
-				dma-coherent;
-				pinctrl-names = "default";
-				status = "okay";
-				cp1_eth0: eth0@010000 {
-					interrupts = <ICU_GRP_NSR 39 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 43 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 47 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 51 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 55 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 129 IRQ_TYPE_LEVEL_HIGH>;
-					interrupt-names = "tx-cpu0", "tx-cpu1", "tx-cpu2",
-							  "tx-cpu3", "rx-shared", "link";
-					mac-address = [00 00 00 00 00 04];
-					port-id = <0>; /* pp2_port_id */
-					gop-port-id = <0>;
-					phy-mode = "sfi";
-					status = "okay";
-				};
-				cp1_eth1: eth1@020000 {
-					interrupts = <ICU_GRP_NSR 40 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 44 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 48 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 52 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 56 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 128 IRQ_TYPE_LEVEL_HIGH>;
-					interrupt-names = "tx-cpu0", "tx-cpu1", "tx-cpu2",
-							  "tx-cpu3", "rx-shared", "link";
-					mac-address = [00 00 00 00 00 05];
-					port-id = <1>; /* pp2_port_id */
-					gop-port-id = <2>;
-					phy-mode = "rgmii-id";
-					phy = <&cp1_phy1>;
-					status = "okay";
-				};
-				cp1_eth2: eth2@030000 {
-					status = "disabled";
-				};
-			};
-			comphy {
-				status = "disabled";
-			};
-			spi1: spi@700680 {
-				status = "okay";
-				pinctrl-names = "default";
-				pinctrl-0 = <&cp1_spi1_pins>;
-			};
-		};
-		pcie0@600000 {
-			status = "okay";
-		};
-		pcie1@620000 {
-			status = "okay";
-		};
-		pcie2@640000 {
-			status = "okay";
-		};
-	};
-};
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts
deleted file mode 100644
index 48e67ed..0000000
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin-pp2-vanilla.dts
+++ /dev/null
@@ -1,321 +0,0 @@
-/*
- * Copyright (C) 2016 Marvell Technology Group Ltd.
- *
- * This file is dual-licensed: you can use it either under the terms
- * of the GPLv2 or the X11 license, at your option. Note that this dual
- * licensing only applies to this file, and not this project as a
- * whole.
- *
- *  a) This library is free software; you can redistribute it and/or
- *     modify it under the terms of the GNU General Public License as
- *     published by the Free Software Foundation; either version 2 of the
- *     License, or (at your option) any later version.
- *
- *     This library is distributed in the hope that it will be useful,
- *     but WITHOUT ANY WARRANTY; without even the implied warranty of
- *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *     GNU General Public License for more details.
- *
- * Or, alternatively,
- *
- *  b) Permission is hereby granted, free of charge, to any person
- *     obtaining a copy of this software and associated documentation
- *     files (the "Software"), to deal in the Software without
- *     restriction, including without limitation the rights to use,
- *     copy, modify, merge, publish, distribute, sublicense, and/or
- *     sell copies of the Software, and to permit persons to whom the
- *     Software is furnished to do so, subject to the following
- *     conditions:
- *
- *     The above copyright notice and this permission notice shall be
- *     included in all copies or substantial portions of the Software.
- *
- *     THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
- *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
- *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
- *     NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
- *     HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
- *     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
- *     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
- *     OTHER DEALINGS IN THE SOFTWARE.
- */
-
-/*
- * Device Tree file for MACCHIATOBin Armada 8040 community board platform
- */
-
-#include "armada-8040.dtsi"
-
-/ {
-	model = "Marvell 8040 MACHIATOBin - pp2-vanilla";
-	compatible = "marvell,armada8040-mcbin", "marvell,armada8040",
-			"marvell,armada-ap806-quad", "marvell,armada-ap806";
-
-	aliases {
-		ethernet0 = &cp0_eth0;
-		ethernet1 = &cp1_eth0;
-		ethernet2 = &cp1_eth1;
-		ethernet3 = &cp1_eth2;
-	};
-
-	memory@00000000 {
-		device_type = "memory";
-		reg = <0x0 0x0 0x0 0x80000000>;
-	};
-
-	ap806 {
-		config-space {
-			serial@512000 {
-				status = "okay";
-			};
-			sdhci@6e0000 {
-				vqmmc-supply = <&vccq_mmc_reg>;
-				bus-width = <8>;
-				no-1-8-v;
-				non-removable;
-				marvell,xenon-emmc;
-				status = "okay";
-			};
-			vccq_mmc_reg: regulator@0 {
-				compatible = "regulator-fixed";
-				regulator-name = "vccq-mmc-reg";
-				regulator-min-microvolt = <3300000>;
-				regulator-max-microvolt = <3300000>;
-				regulator-always-on;
-				status = "okay";
-			};
-		};
-	};
-
-	cp0 {
-		config-space {
-			/delete-node/ppv22@000000;
-			pinctrl@440000 {
-				xhci0_vbus_pins: xhci0-vbus-pins {
-					marvell,pins = "mpp47";
-					marvell,function = "gpio";
-				};
-				pcie_reset_pins: pcie-reset-pins {
-					marvell,pins = "mpp52";
-					marvell,function = "gpio";
-				};
-			};
-			cp0_comphy: comphy {
-				status = "disabled";
-			};
-			sdhci@780000 {
-				vqmmc-supply = <&vccq_sdio_reg>;
-				bus-width = <4>;
-				no-1-8-v;
-				broken-cd;
-				pinctrl-names = "default";
-				pinctrl-0 = <&cp0_sdhci_pins>;
-				status = "okay";
-			};
-			cp0_i2c0: i2c@701000 {
-				clock-frequency = <100000>;
-				pinctrl-names = "default";
-				pinctrl-0 = <&cp0_i2c0_pins>;
-				status = "okay";
-			};
-			cp0_sata: sata@540000 {
-				status = "okay";
-
-				sata-port@0 {
-					status = "disabled";
-				};
-				sata-port@1 {
-					status = "okay";
-				};
-			};
-			mdio@12a200 {
-				#address-cells = <1>;
-				#size-cells = <0>;
-				status = "okay";
-				smi_phy0: ethernet-phy@0 {
-					reg = <0>;
-				};
-			};
-			mdio@12a600 {
-				#address-cells = <1>;
-				#size-cells = <0>;
-				status = "okay";
-				cp0_phy0: ethernet-phy@0 {
-					compatible = "ethernet-phy-ieee802.3-c45";
-					reg = <0>;
-				};
-				cp0_phy1: ethernet-phy@1 {
-					compatible = "ethernet-phy-ieee802.3-c45";
-					reg = <8>;
-				};
-			};
-			ppv22_vanilla@000000 {
-				compatible = "marvell,armada-7k-pp22";
-				reg = <0x0 0x100000>, <0x129000 0xb000>;
-				clocks = <&cp0_syscon0 1 3>, <&cp0_syscon0 1 9>, <&cp0_syscon0 1 5>;
-				clock-names = "pp_clk", "gop_clk", "mg_clk";
-				dma-coherent;
-				pinctrl-names = "default";
-				status = "okay";
-				cp0_eth0: eth0@010000 {
-					interrupts = <ICU_GRP_NSR 39 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 43 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 47 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 51 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 55 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 129 IRQ_TYPE_LEVEL_HIGH>;
-					interrupt-names = "tx-cpu0", "tx-cpu1", "tx-cpu2",
-							  "tx-cpu3", "rx-shared", "link";
-					mac-address = [00 00 00 00 00 01];
-					port-id = <0>; /* pp2_port_id */
-					gop-port-id = <0>;
-					phy-mode = "sfi"; /* lane-2 */
-					phy = <&cp0_phy0>;
-					status = "okay";
-				};
-				cp0_eth1: eth1@020000 {
-					interrupts = <ICU_GRP_NSR 40 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 44 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 48 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 52 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 56 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 128 IRQ_TYPE_LEVEL_HIGH>;
-					interrupt-names = "tx-cpu0", "tx-cpu1", "tx-cpu2",
-							  "tx-cpu3", "rx-shared", "link";
-					mac-address = [00 00 00 00 00 02];
-					port-id = <1>; /* pp2_port_id */
-					gop-port-id = <2>;
-					status = "disabled";
-				};
-				cp0_eth2: eth2@030000 {
-					interrupts = <ICU_GRP_NSR 41 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 45 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 49 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 53 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 57 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 127 IRQ_TYPE_LEVEL_HIGH>;
-					interrupt-names = "tx-cpu0", "tx-cpu1", "tx-cpu2",
-							  "tx-cpu3", "rx-shared", "link";
-					mac-address = [00 00 00 00 00 03];
-					port-id = <2>; /* pp2_port_id */
-					gop-port-id = <3>;
-					status = "disabled";
-				};
-			};
-		};
-		pcie0@600000 {
-			num-lanes = <4>;
-			status = "okay";
-			reset-gpios = <&cp0_gpio1 20 GPIO_ACTIVE_HIGH>;
-			phys = <&cp0_comphy 0 COMPHY_PCIE0
-				&cp0_comphy 1 COMPHY_PCIE0
-				&cp0_comphy 2 COMPHY_PCIE0
-				&cp0_comphy 3 COMPHY_PCIE0
-				>;
-			phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
-			pinctrl-names = "default";
-			pinctrl-0 = <&pcie_reset_pins>;
-		};
-		reg_usb3h0_vbus: usb3-vbus0 {
-			compatible = "regulator-fixed";
-			pinctrl-names = "default";
-			pinctrl-0 = <&xhci0_vbus_pins>;
-			regulator-name = "reg-usb3h0-vbus";
-			regulator-min-microvolt = <5000000>;
-			regulator-max-microvolt = <5000000>;
-			enable-active-high;
-			gpio = <&cp0_gpio1 15 GPIO_ACTIVE_HIGH>;
-			status = "okay";
-		};
-		vccq_sdio_reg: regulator@1 {
-			compatible = "regulator-fixed";
-			regulator-name = "vccq-sdio-reg";
-			regulator-min-microvolt = <3300000>;
-			regulator-max-microvolt = <3300000>;
-			regulator-always-on;
-			status = "okay";
-		};
-	};
-
-	cp1 {
-		config-space {
-			/delete-node/ppv22@000000;
-			cp1_comphy1: comphy {
-				status = "disabled";
-			};
-			cp1_usb3h0: usb3@500000 {
-				usb-phy = <&cp1_usb3h0_phy>;
-				status = "okay";
-			};
-			cp1_usb3h0_phy: usb3_phy0 {
-				compatible = "usb-nop-xceiv";
-				vcc-supply = <&reg_usb3h0_vbus>;
-			};
-			cp1_sata: sata@540000 {
-				status = "okay";
-
-				sata-port@0 {
-					status = "okay";
-				};
-				sata-port@1 {
-					status = "okay";
-				};
-			};
-			ppv22_vanilla@000000 {
-				compatible = "marvell,armada-7k-pp22";
-				reg = <0x0 0x100000>, <0x129000 0xb000>;
-				clocks = <&cp1_syscon0 1 3>, <&cp1_syscon0 1 9>, <&cp1_syscon0 1 5>;
-				clock-names = "pp_clk", "gop_clk", "mg_clk";
-				dma-coherent;
-				pinctrl-names = "default";
-				status = "okay";
-				cp1_eth0: eth0@010000 {
-					phy-mode = "sfi";
-					phy = <&cp0_phy1>;
-					port-id = <0>; /* pp2_port_id */
-					gop-port-id = <0>;
-					interrupts = <ICU_GRP_NSR 39 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 43 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 47 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 51 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 55 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 129 IRQ_TYPE_LEVEL_HIGH>;
-					interrupt-names = "tx-cpu0", "tx-cpu1", "tx-cpu2",
-							  "tx-cpu3", "rx-shared", "link";
-					mac-address = [00 00 00 00 00 04];
-					status = "okay";
-				};
-				cp1_eth1: eth1@020000 {
-					phy-mode = "sgmii";
-					port-id = <1>; /* pp2_port_id */
-					phy = <&smi_phy0>;
-					gop-port-id = <2>;
-					interrupts = <ICU_GRP_NSR 40 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 44 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 48 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 52 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 56 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 128 IRQ_TYPE_LEVEL_HIGH>;
-					interrupt-names = "tx-cpu0", "tx-cpu1", "tx-cpu2",
-							  "tx-cpu3", "rx-shared", "link";
-					mac-address = [00 00 00 00 00 05];
-					status = "okay";
-				};
-				cp1_eth2: eth2@030000 {
-					interrupts = <ICU_GRP_NSR 41 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 45 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 49 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 53 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 57 IRQ_TYPE_LEVEL_HIGH>,
-						     <ICU_GRP_NSR 127 IRQ_TYPE_LEVEL_HIGH>;
-					interrupt-names = "tx-cpu0", "tx-cpu1", "tx-cpu2",
-							  "tx-cpu3", "rx-shared", "link";
-					mac-address = [00 00 00 00 00 06];
-					port-id = <2>;
-					gop-port-id = <3>;
-					status = "disabled";
-				};
-			};
-		};
-	};
-};
diff --git a/tools/configs/pp2-vanilla b/tools/configs/pp2-vanilla
deleted file mode 100644
index 1c78c70..0000000
--- a/tools/configs/pp2-vanilla
+++ /dev/null
@@ -1,2 +0,0 @@
-CONFIG_MVPP2=y
-CONFIG_MVPP2X=n
-- 
2.7.4

